### ISSUE TITLE
0.2.47 release was not completed

### ISSUE THREAD

**Issue by chenrui333** [nektos/act#1894]
👋 it looks like 0.2.47 release was not completed, see [this action run](https://github.com/nektos/act/actions/runs/5428652209/jobs/9873037000), raise this issue for some awareness. Thanks!

relates to https://github.com/Homebrew/homebrew-core/pull/135514

**Comment by cplee** [comment]
Thanks @chenrui333  - i'll rerun the release once #1895 is merged

**Comment by chenrui333** [comment]
Thanks, trying in [the new PR](https://github.com/Homebrew/homebrew-core/pull/136277)
